BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -445,8 +440,9 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         // After a rotation or other size change the optimal width for our cells may have changed.
         // We instruct the collectionView to reload so widths are recalculated.
-        coordinator.animate { _ in } completion: { _ in
-            self.collectionView?.reloadData()
+        coordinator.animate { _ in } completion: { [weak self] _ in
```

Comment:
Could you explain why you chose to use the `weak self` pattern here? My impression is the block passed to `animate` is short lived.

Reply:
the layout changed, not the data

I have fixed this by indicating that the layout has become invalid.